testsuite/features/secondary/srv_task_status_engine.feature

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Task Engine Status
+
+  Scenario: Login as admin
+    Given I am authorized for the "Admin" section
+    
+  Scenario: Check if the Task Engine Status page exists
+    When I follow the left menu "Admin > Task Engine Status > Last Execution Times"
+    Then I should see a "Task Engine Status" text
+    And I should see a "The following is a status report for the various tasks run by the Uyuni task engine:" text
+    And I should see a "Runtime Status" text
+    And I should see a "Last Execution Times" link in the left menu
+    And I should see a "Runtime Status" link in the left menu
+
+  Scenario: Check if the Runtime Status Page exists
+    When I follow the left menu "Admin > Task Engine Status > Runtime Status"
+    Then I should see a "Task Engine Status" text
+    And I should see a "Last Execution Times" text
+    And I should see a "The server is running or has finished executing the following tasks during the latest 5 minutes." text
+    And I should see a "Last Execution Times" link in the left menu
+    And I should see a "Runtime Status" link in the left menu
+  
+  Scenario: Run a remote command on a minion to have and check if it shows up on Last Execution Times page
+    When I follow the left menu "Admin > Task Engine Status > Last Execution Times"
+    And I run "cobbler sync" on "server"
+    And I refresh the page
+    Then I should see a "Cobbler Sync:" text
+    And I should see the correct timestamp for task "Cobbler Sync:"
+    And I should see a "FINISHED" text
+
+  Scenario: Resync a product to trigger a new tast and check if it is visible on the Runtime Status page
+    When I follow the left menu "Admin > Task Engine Status > Runtime Status"
+    And I follow the left menu "Admin > Setup Wizard > Products"
+    And I wait until I do not see "Loading" text
+    And I enter "SUSE Linux Enterprise Server 15 SP4 x86_64" as "product-description-filter"
+    And I wait until I see "SUSE Linux Enterprise Server 15 SP4 x86_64" text
+    Then I should see the "SUSE Linux Enterprise Server 15 SP4 x86_64" selected
+    When I click on "Schedule channels product resync"
+    And I follow the left menu "Admin > Task Engine Status > Runtime Status"
+    Then I wait until I see "repo-sync" text
```

Comment:
I prefer "And" here, and "Then" on next line. Maybe a matter of tastes.

Reply:
you're right it does make it a little clearer 👍🏼